commands/registry.go

```diff
@@ -259,9 +329,123 @@ func RunRegistryLogout(c *CmdConfig) error {
 	return cmd.Run()
 }
 
+// Repository Run Commands
+
+// RunListRepositories lists repositories for the registry
+func RunListRepositories(c *CmdConfig) error {
+	registry, err := c.Registry().Get()
+	if err != nil {
+		return fmt.Errorf("failed to get registry: %w", err)
+	}
+
+	repositories, err := c.Registry().ListRepositories(registry.Name)
+	if err != nil {
+		return err
+	}
+
+	return displayRepositories(c, repositories...)
+}
+
+// RunListRepositoryTags lists tags for the repository in a registry
+func RunListRepositoryTags(c *CmdConfig) error {
+	if len(c.Args) != 1 {
+		return doctl.NewMissingArgsErr(c.NS)
+	}
+
+	registry, err := c.Registry().Get()
+	if err != nil {
+		return fmt.Errorf("failed to get registry: %w", err)
+	}
+
+	tags, err := c.Registry().ListRepositoryTags(registry.Name, c.Args[0])
+	if err != nil {
+		return err
+	}
+
+	return displayRepositoryTags(c, tags...)
+}
+
+// RunRepositoryDeleteTag deletes one or more repository tags
+func RunRepositoryDeleteTag(c *CmdConfig) error {
+	force, err := c.Doit.GetBool(c.NS, doctl.ArgForce)
+	if err != nil {
+		return err
+	}
+
+	if len(c.Args) < 2 {
+		return doctl.NewMissingArgsErr(c.NS)
+	}
+
+	registry, err := c.Registry().Get()
+	if err != nil {
+		return fmt.Errorf("failed to get registry: %w", err)
+	}
+
+	repository := c.Args[0]
+	tags := c.Args[1:]
+
+	if !force && AskForConfirm(fmt.Sprintf("delete %d repository tag(s)", len(tags))) != nil {
+		return fmt.Errorf("operation aborted")
+	}
+
+	for _, tag := range tags {
+		if err := c.Registry().DeleteTag(registry.Name, repository, tag); err != nil {
+			return err
```

Comment:
Might be more user-friendly to aggregate errors and return them at the end, so that one deletion failing in the middle of a larger set of tags doesn't block deleting the rest. Not sure if there's any pattern/precedent to follow from other subcommands.

Reply:
I had considered this when initially implementing but after a quick (randomized) survey of other commands and deletion functionality, they all exit on first encounter of a failure without processing the rest.

Thoughts @hilary ?

Reply:
How likely is it that a deletion will fail? If it's pretty rare, I think it's okay to keep this as-is.

Reply:
Deletion failure would be pretty rare. 

I agree the experience would be a little nicer not blocking on failures. Since it sounds like we are okay with that process, I opted to update.